aslam_cv_cameras/include/aslam/cameras/undistorter-mapped.h

```diff
@@ -69,13 +69,21 @@ class MappedUndistorter : public Undistorter {
   /// \param[in] interpolation Interpolation method used for undistortion.
   ///                          (\ref InterpolationMethod)
   MappedUndistorter(aslam::Camera::Ptr input_camera, aslam::Camera::Ptr output_camera,
-                    const cv::Mat& map_u, const cv::Mat& map_v, InterpolationMethod interpolation);
+                   const cv::Mat& map_u, const cv::Mat& map_v, 
+                   const cv::Mat& map_u_float, const cv::Mat& map_v_float, 
```

Comment:
why does this now take both the fixed point maps and the float map? Shouldn't it only need one of the two?